### PR TITLE
Meta: Automatically create GitHub Releases with changelog 

### PR DESCRIPTION
**Pull request by notlmn**
Fixes https://github.com/sindresorhus/refined-github/issues/3198

Use https://github.com/marketplace/actions/release-with-changelog for creating release notes for every tag we create (which is also automated).

I agree that it's not perfect, but feedback welcome.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - "23.*"
+      - "24.*"
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 100
```
Comment (fregante) [inline .github/workflows/release.yml:18]:
```suggestion
        fetch-depth: 20
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    - uses: notlmn/release-with-changelog@v1
+      with:
+        include-hash: true
+        include-range: true
```
Comment (fregante) [inline .github/workflows/release.yml:22]:
```suggestion
      with:
        include-range: false
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "20.*"
+      - "21.*"
+      - "22.*"
+      - "23.*"
+      - "24.*"
```
Comment (sindresorhus) [inline .github/workflows/release.yml:10]: Single-quotes

**Comment by fregante** [comment]
@notlmn can you finish and merge this like https://github.com/notlmn/browser-extension-template/pull/38 before the automatic deployment tomorrow?

**Comment by fregante** [comment]
I think you might have pushed the `hotfix` tag again. Please run this to delete the tags from your fork and locally, we keep pulling them from old forks and pushing some troublesome ones back to this repo

```sh
git tag -l | xargs git tag -d
git ls-remote --tags --refs origin | cut -f2 | xargs git push origin --delete
```

**Comment by fregante** [comment]
Just in time for deployment 🎉

**Comment by yakov116** [comment]
@fregante chrome did not release yet (first time in a long time that its taking more than 15 min) anything to do with this? or there is no connection?

**Comment by darkred** [comment]
> taking more than 15 min

It's been 1 hour since the 20.11.19 release, and in CWS it's still the previous version, 20.11.12.
I see that the relevant Submit action has failed for Chrome: 
https://github.com/sindresorhus/refined-github/runs/1425836312?check_suite_focus=true


> "message": "Publish condition not met: To publish your item, you must certify that your data usage complies with our Developer Program Policies. You can certify this on the Privacy tab of the item edit page of the Developer Dashboard.",

---

In AMO it has the latest, 20.11.19 .

**Pull request merged**